dashboard/lib/policies/child_account.rb

```diff
@@ -1,21 +1,25 @@
+require 'cdo/shared_constants'
+
 class Policies::ChildAccount
   # Values for the `child_account_compliance_state` attribute
   module ComplianceState
     # The student's account has been used to issue a request to a parent.
-    LOCKED_OUT = 'l'.freeze
+    LOCKED_OUT = SharedConstants::CHILD_ACCOUNT_COMPLIANCE_STATES.LOCKED_OUT
 
     # The student's account has been used to issue a request to a parent.
-    REQUEST_SENT = 's'.freeze
+    REQUEST_SENT = SharedConstants::CHILD_ACCOUNT_COMPLIANCE_STATES.REQUEST_SENT
 
     # The student's account has been approved by their parent.
-    PERMISSION_GRANTED = 'g'.freeze
+    PERMISSION_GRANTED = SharedConstants::CHILD_ACCOUNT_COMPLIANCE_STATES.PERMISSION_GRANTED
   end
 
   # The individual US State child account policy configuration
   # max_age: the oldest age of the child at which this policy applies.
+  # start_date: the date on which this policy first went into effect.
   STATE_POLICY = {
     'CO' => {
-      max_age: 12
+      max_age: 12,
+      start_date: Date.new(2023, 7, 1)
```

Comment:
**Important change request**
The scheduling of policies is managed in the `cpa_schedule` DCDO config. See  `dashboard/lib/cpa.rb`.

Reply:
Good catch. Changed this to retrieve the DCDO value, but with a hardcoded default as a fallback. Felt better in case the code is executed in an environment where the schedule isn't set, but still allows for potential overrides via the DCDO value. The alternative would be to risk having the value be nil, and have to stub the value in all our tests.